README.md

```diff
@@ -664,6 +664,7 @@ See https://github.com/n1trux/awesome-sysadmin#identity-management
 
 _RSS/Atom automation_
 
+- [Full-Text RSS](https://fivefilters.org/content-only) - Extract article content from news sites and blogs and convert RSS feeds that contain only extracts of stories to full-text feeds. Developed by FiveFilters.org. ([Source](https://bitbucket.org/fivefilters/full-text-rss)) `GPL-3.0` `PHP`
```

Comment:
The correct syntax is `[Source Code](https://bitbucket.org/fivefilters/full-text-rss)`

Reply:
Done